src/modules/main/Main.tsx

```diff
@@ -29,10 +34,18 @@ const Main = ({ match }: RouteChildrenProps) => {
 
       <main>
         <Switch>
+          <Route
+            exact
+            path={`${match.path}/admin`}
+            render={() => <Redirect to={`${match.path}/admin/${AdminTabCategory.TECHNOLOGIES}`} />}
+          />
+
           <Route exact path={`${match.path}/dashboard`} render={() => <div>dashboard</div>} />
 
           <Route exact path={`${match.path}/projects`} render={() => <div>projects</div>} />
 
+          <Route path={`${match.path}/admin/:category`} component={AdminPanel} />
```

Comment:
Instead of putting here /:category you should add /admin/dictionaries. Imagine if we add next admin route we will be blocked by this implementation